rules/windows/process_creation/proc_creation_win_whoami_priv_discovery.yml

```diff
@@ -25,4 +25,4 @@ detection:
     condition: all of selection_*
 falsepositives:
     - Unknown
-level: high
+level: medium
```

Comment:
I am interested in the reasoning here as well.

Reply:
Discussed in private